### PR TITLE
Minor fixes to metasignatures

### PR DESCRIPTION
**Pull request by pawel-czyz**
This PR fixes some minor issues with the metasignatures code and asks some questions (the method is sophisticated enough that I didn't follow).

### CODE REVIEW

File: src/cansig/metasignatures/clustering.py [hunk 1]
```diff
@@ -313,6 +313,7 @@ def get_final_clustering_jaccard(
         )
         return cleaned_clusters
 
+    # TODO(Josephine, Pawel): This recursion potentially never finishes.
```
Comment (josephineyates) [inline src/cansig/metasignatures/clustering.py:316]: I agree, I think just adding a 
to the `if correlation_iteration `
`if correlation_iteration or n_clusters>20:`
would be sufficient. What do you think?

File: src/cansig/run/metasignatures.py [hunk 2]
```diff
@@ -45,17 +45,19 @@ def parse_args():
...
     )
 
     parser.add_argument(
         "--linkage",
         type=str,
-        choice=["ward", "average", "single", "complete", "weighted", "centroid", "median"],
+        choices=["ward", "average", "single", "complete", "weighted", "centroid", "median"],
         help="the linkage used for the agglomerative clustering for metasignatures",
-        default="jaccard",
+        # TODO(Josephine, Pawel): Is "jaccard" allowed? Should it be the default?
```
Comment (josephineyates) [inline src/cansig/run/metasignatures.py:58]: Hi Pawel, I am seeing that there has been some versioning weirdness going on, I had corrected both of these already. Good catch though! No the default should be average, jaccard is not allowed here as you pointed out!

**Review by josephineyates — APPROVED** [review]
Good catches. Some suggestions in the comments, then if it runs we are good to go I think :)

File: src/cansig/run/metasignatures.py (hunk 2 above)
Comment (pawel-czyz) [reply src/cansig/run/metasignatures.py]: Thanks, changed the default!

**Pull request merged**